lib/internal/per_context/primordials.js

```diff
@@ -129,20 +144,21 @@ function makeSafe(unsafe, safe) {
             return new SafeIterator(this);
           };
         }
-        Reflect.defineProperty(safe.prototype, key, desc);
+        ReflectDefineProperty(safe.prototype, key, desc);
       }
     }
   } else {
     copyProps(unsafe.prototype, safe.prototype);
   }
   copyProps(unsafe, safe);
 
-  Object.setPrototypeOf(safe.prototype, null);
-  Object.freeze(safe.prototype);
-  Object.freeze(safe);
+  ObjectSetPrototypeOf(safe.prototype, null);
+  ObjectFreeze(safe.prototype);
+  ObjectFreeze(safe);
   return safe;
 }
 primordials.makeSafe = makeSafe;
+/* eslint-disable node-core/prefer-primordials */
```

Comment:
nit: add a line break so it's separated from the `makeSafe` declaration.
```suggestion

/* eslint-disable node-core/prefer-primordials */
```